src/features/island/hud/components/PowerSkillsButton.tsx

```diff
@@ -3,9 +3,44 @@ import lightning from "assets/icons/lightning.png";
 import { PIXEL_SCALE } from "features/game/lib/constants";
 import { SUNNYSIDE } from "assets/sunnyside";
 import { PowerSkills } from "./PowerSkills";
+import { useGame } from "features/game/GameProvider";
+import {
+  BumpkinSkillRevamp,
+  getPowerSkills,
+} from "features/game/types/bumpkinSkills";
+import { BumpkinRevampSkillName } from "features/game/types/bumpkinSkills";
+import { useSelector } from "@xstate/react";
 
 export const PowerSkillsButton: React.FC = () => {
   const [show, setShow] = useState(false);
+  const { gameService } = useGame();
```

Comment:
I would suggest using the other `useContext(GameContext)` approach to get the `gameService`

I believe `useGame` is used for re-renders (whenever anything in the game changes). This may cause unnecessary re-renders